Sources/NIOSSL/NIOSSLHandler.swift

```diff
@@ -241,19 +249,19 @@ public class NIOSSLHandler : ChannelInboundHandler, ChannelOutboundHandler, Remo
                 channelClose(context: context, reason: error)
                 return
             }
+            
+            if let additionalCertificateChainVerification = connection.parentContext.additionalCertificateChainVerification {
+                state = .additionalVerification
+                additionalCertificateChainVerification(context.channel)
+                    .hop(to: context.eventLoop)
+                    .whenComplete { result in
+                        self.completedAdditionalCertificateChainVerification(result: result, context: context)
```

Comment:
As we store the `context` on `self`, we don't need to close over it here, which will make this closure "free" to allocate (as it will only close over a single `class`).